components/expense/expense.js

```diff
@@ -0,0 +1,110 @@
+import {createElement} from "../../commons.js";
+import { tableConfig } from '../../table_config.js';
+import { Balance } from '../balance/balance.js';
+
+const EXPENSEICON = "../../icons/credit_card-24px.svg";
+const EXPENSE_VALUE_LOGO = "../../icons/attach_money-24px.svg";
+
+export class Expense {
+
+    constructor(selector, expenses) {
+        this.element = document.querySelector('.container');
+        this.expenses = expenses;
+        this.selector = selector;
+        this.expenseComponent = createElement("div",{class: selector});
+        this.table = createElement("table", {class: "expenses-table"});
+        this.createExpenseComponent();
+        this.createHeader();
+        this.createBody();
+    }
+
+
+   createExpenseComponent() {
+        const title = createElement("span", {class:"expense_title"});
+        let sumOfExpenses = 0;
+
+        title.innerHTML=this.selector;
+        this.expenseComponent.append(title);
+
+        const expenseIcon = createElement("img", {class:"expense_icon material-icons", src: EXPENSEICON});
+        this.expenseComponent.append(expenseIcon);
+
+        const expenseValueLogo = createElement("img", {class: "expense_value_logo", src: EXPENSE_VALUE_LOGO});
+
+        const expenseValue = createElement("div", {class:"expense_value"});
+        const BudgetAndExpensesComponent = document.querySelector(".Budget_And_Expenses");
+
+         this.expenses.forEach((item) => {
+             sumOfExpenses = sumOfExpenses + parseInt(item.expense_Value);
+         });
```

Comment:
please consider using reduce instead. 

`
  const sumOfExpenses  = this.expenses.reduce((acc,item)=> acc+ parseInt(item.expense_Value), 0);
`